script/DEV-004_ad_create.sql

```diff
@@ -10,6 +12,8 @@ CREATE TABLE public.ad
     phone character varying COLLATE pg_catalog."default",
     email character varying COLLATE pg_catalog."default",
     rubric integer,
+    addate date,
```

Comment:
No, here you should use another sql instead adding to create table script:
ALTER TABLE ad ADD COLUMN date TIMESTAMP;

Reply:
done